### DISCUSSION TITLE
✅ 🚀 Feature Request: Aspect Ratio and Object Fit options for Image Module - [ED-10423]

### DISCUSSION THREAD

**Discussion by masondigital-jordan** [elementor/elementor#21756] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I often need images to be of a certain size or aspect ratio in our designs. This usually can be done using background images, but in many cases it's nice to use an actual <img> with alt tags for SEO purposes. So I usually use aspect-ratio and object-fit/object-position to achieve a similar look as if using a background image. In Elementor I can only do this with an image module by adding some cusom css

### Describe the solution you'd like

It would be awesome if the image module could have an option to select/set an aspect ratio, as well as object fit settings (mainly cover or contain) along with object-position settings to set the position. This could also be achieved with a focal point picker!

### Describe alternatives you've considered

It's fine to use css for now, but our clients often aren't css savvy when we hand off the project. If these options could be built into the interface, it would be easier for more users to use!

### Additional context

BTW, we love Elementor :)

**Comment by nicholaszein** [comment]
Hello @masondigital-jordan,

thanks for submitting your feature request! 🙏 

Part of your request is already possible with the Image Widget. All you have to do is to set a height, and an option to choose `object-fit` will appear.

As for `object-position`, we are considering adding it to Elementor, keep on eye on the Announcements Category for new Announcements. 😉 

Best regards

**Reply by nicholaszein** [reply]
Also, please refer to the documentation:

https://elementor.com/help/image-widget/

**Reply by mapsmarketing** [reply]
@nicholaszein 

Would be great to have since the focal point of images varies and setting `object-fit` to `cover` doesn't always show the part of the image that's desired. Considering the dropdown already exists for background images, I doubt it would take much time to add another dropdown for the image widget for generic and custom positions when there's a height set for it.

Are pull requests welcomed for feature additions such as this?

edit: just forked the main branch and realised you boys have already added it :)

**Comment by nicholaszein** [comment]
Hello, there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.14.0! 🥳 

You can now set `Object Position` when the `Height` and `Object Fit` controls are customized on image widgets.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
